src/ChunkMap.cpp

```diff
@@ -2593,6 +2593,34 @@ bool cChunkMap::ForEachChunkInRect(int a_MinChunkX, int a_MaxChunkX, int a_MinCh
 
 
 
+bool cChunkMap::ForEachLoadedChunk(cChunkDataCallback & a_Callback)
```

Comment:
What's the point of Result in this function? It just seems to be an alias for true to me.